CONTRIBUTING.md

```diff
@@ -1,123 +1,126 @@
-
-Majority of NEAR developers use CLion with Rust plugin as their primary IDE.
-
-We also had success with VSCode with rust-analyzer, see the steps for installation [here](https://commonwealth.im/near/proposal/discussion/338-remote-development-with-vscode-and-rustanalyzer).
-
-Some of us use VIM with [rust.vim](https://github.com/rust-lang/rust.vim) and [rusty-tags](https://github.com/dan-t/rusty-tags). It has fewer features than CLion or VSCode, but overall provides a usable setting.
-
-Refer to [this document](https://docs.nearprotocol.com/docs/contribution/nearcore) for details on setting up your environment.
-
-# Release Schedule
-
-Once your change ends up in master, it will be released with the rest of the changes by other contributors on the regular release schedules.
-
-On [betanet](https://docs.near.org/docs/concepts/networks#betanet) we run nightly build from master with all the nightly protocol feature enabled.
-Every five weeks, we stabilize some protocol features and make a release candidate for testnet.
-The process for feature stabilization can be found in [this document](docs/protocol_upgrade.md).
-After the release candidate has been running on testnet for four weeks and no issue is observed,
-we stabilize and publish the release for mainnet.
-
-# Crate Versioning and Publishing
-
-While all the crates in the workspace are directly unversioned (`v0.0.0`), they all share a unified variable version in the [workspace manifest](Cargo.toml). This keeps versions consistent across the workspace and informs their versions at the moment of publishing.
-
-We also have CI infrastructure set up to automate the publishing process to crates.io. So, on every merge to master, if there's a version change, it's automatically applied to all the crates in the workspace and it attempts to publish the new versions of all non-private crates. All crates that should be exempt from this process should be marked `private`. That is, they should have the `publish = false` specification in their package manifest.
-
-This process is managed by [cargo-workspaces](https://github.com/pksunkara/cargo-workspaces), with a [bit of magic](https://github.com/pksunkara/cargo-workspaces/compare/master...miraclx:grouping-and-exclusion#files_bucket) sprinkled on top.
-
-## Issue Labels
-
-Issue labels are of the following format `<type>-<content>` where `<type>` is a capital letter indicating the type of the label and `<content>` is a hyphened phrase indicating what is label is about.
-For example, in the label `C-bug`, `C` means category and `bug` means that the label is about bugs.
-Common types include `C`, which means category, `A`, which means area, `T`, which means team.
-
-An issue can have multiple labels including which area it touches, which team should be responsible for the issue, and so on.
-Each issue should have at least one label attached to it after it is triaged and the label could be a general one, such as `C-enhancement` or `C-bug`.
```

Comment:
These two sections felt a bit out of place in this file so I dropped them.  I am happy to include them if someone feels strongly that they should be included here.

Reply:
:+1: for trimming this out of CONTRIBUTING.md. CONTRIBUTING.md is a landing page, it should be short and to the point, and just link other documents. 

:-1: for removing this info -- this documents non-obvious institutional knowledge, it should be documented somewhere. It's probably best to find a place for it somewhere in the `docs/` folder. There isn't an obvious place to put it, so I suggest creating a `docs/README.md`. That'll get automatically displayed by github when opening a folder, and can serve both as an index and as dumping ground for stuff we are not quite sure where else to put.

Reply:
Moved to `docs/README.md`